scalameta/contrib/src/main/scala/scala/meta/contrib/implicits/implicits.scala

```diff
@@ -0,0 +1,3 @@
+package scala.meta.contrib.implicits
+
+trait implicits extends Statements.ToStatementOps
```

Comment:
Let's discuss the two approaches. I understand the theoretical appeal of modularizing things, but in my opinion the overhead of requiring two (or more, if we have multiple modules) imports is significant. Could you guys elaborate on what the modularization buys us?